src/corecel/math/Turn.hh

```diff
@@ -35,7 +35,13 @@ struct HalfPi
 };
 
 //---------------------------------------------------------------------------//
-//! Quantity denoting a full turn
+/*!
+ * Quantity denoting a full turn.
+ *
+ * Turns are a useful way of representing angles without the historical
```

Comment:
This is a good improvement.  Is it related to the issue of this PR?

Reply:
Yeah, this is all about sincospi which lets you use rational values for turns.